project/SparkBuild.scala

```diff
@@ -486,9 +485,11 @@ object SparkParallelTestGrouping {
   )
 
   private val DEFAULT_TEST_GROUP = "default_test_group"
+  private val HIVE_EXECUTION_TEST_GROUP = "hive_execution_test_group"
 
   private def testNameToTestGroup(name: String): String = name match {
     case _ if testsWhichShouldRunInTheirOwnDedicatedJvm.contains(name) => name
+    case _ if name.contains("org.apache.spark.sql.hive.execution") => HIVE_EXECUTION_TEST_GROUP
```

Comment:
@xuanyuanking, shell we add a comment to distinguish `testsWhichShouldRunInTheirOwnDedicatedJvm` and  `org.apache.spark.sql.hive.execution`?

I think the point is that it works differently from other opt-in test cases above defined at `testsWhichShouldRunInTheirOwnDedicatedJvm` since we're grouping all `org.apache.spark.sql.hive.execution.*` into a single group unlike other test cases above, which might be difficult to catch at a glance.

Reply:
Agree, let me add a comment to illustrate.

Reply:
Nit: add comment to mention that this is for reducing the testing time.